qa/tasks/cephadm.py

```diff
@@ -156,14 +156,15 @@ def download_cephadm(ctx, config, ref):
         else:
             ctx.cluster.run(
                 args=[
-                    'git', 'archive',
-                    '--remote=' + git_url,
-                    ref,
-                    'src/cephadm/cephadm',
-                    run.Raw('|'),
-                    'tar', '-xO', 'src/cephadm/cephadm',
-                    run.Raw('>'),
-                    ctx.cephadm,
+                    'git', 'clone', git_url, 'testrepo',
+                    run.Raw('&&'),
+                    'cd', 'testrepo',
+                    run.Raw('&&'),
+                    'git', 'checkout', '-b', 'testbranch', ref,
+                    run.Raw('&&'),
+                    'cp', 'src/cephadm/cephadm', ctx.cephadm,
+                    run.Raw('&&'),
+                    'ls', '-l', ctx.cephadm,
```

Comment:
ni: I understand that with `ls -l` you are testing if the file `ctx.cephadm` exsits or not. This can be achieved by `test -f file_path` without producing any extra output as `ls -l` may do.

Reply:
I actually like that it outputs just so there's something in the log. We do the same thing when curl is used right above this.